src/platform/mock-local-home-platform.ts

```diff
@@ -1,144 +1,91 @@
+export const ERROR_LISTEN_NOT_CALLED: string =
+  'Cannot trigger IdentifyRequest: listen() was not called';
+export const ERROR_UNDEFINED_IDENTIFYHANDLER: string =
+  'Cannot trigger IdentifyRequest: No Identify handler has been set by the fulfillment app';
+export const ERROR_UNDEFINED_VERIFICATIONID: string =
+  'The handler returned an IdentifyResponse with an undefined verificationId';
 
 // TODO(cjdaly): add other radio scan support
-export class MockLocalHomePlatform implements MockUDPListener {
+export class MockLocalHomePlatform {
   //  Singleton instance
   private static instance: MockLocalHomePlatform;
 
-  private udpScanConfigs: UDPScanConfig[] = [];
-  private mockNetwork: MockNetwork | undefined;
   private deviceManager: smarthome.DeviceManager = new DeviceManagerStub();
   private app: AppStub | undefined;
   private localDeviceIds: Map<string, string> = new Map<string, string>();
   private newDeviceRegisteredActions: ((localDeviceId: string) => void)[] = [];
-  private homeAppReady: boolean = false;
 
   private constructor() {}
 
-  public initializeRadio(
-    mockNetwork: MockNetwork,
-    udpScanConfigs: UDPScanConfig[]
-  ) {
-    this.mockNetwork = mockNetwork;
-    this.udpScanConfigs = udpScanConfigs;
-    this.setupUDP();
-  }
-
   public setApp(app: AppStub) {
     this.app = app;
   }
 
-  private onNewDeviceIdRegistered(localDeviceId: string) {
-    this.newDeviceRegisteredActions.forEach(newDeviceRegisteredAction => {
-      newDeviceRegisteredAction(localDeviceId);
-    });
-  }
-
-  public async getNextDeviceIdRegistered(): Promise<string> {
-    return new Promise(resolve => {
-      this.newDeviceRegisteredActions.push(localDeviceId => {
-        resolve(localDeviceId);
-      });
-    });
-  }
-
-  public isHomeAppReady(): boolean {
-    return this.homeAppReady;
+  public getDeviceManager(): smarthome.DeviceManager {
+    return this.deviceManager;
   }
 
-  public notifyHomeAppReady(): void {
-    this.homeAppReady = true;
+  public getLocalDeviceIdMap(): Map<string, string> {
+    return this.localDeviceIds;
   }
 
-  //  Singleton getter
-  public static getInstance(): MockLocalHomePlatform {
-    if (!MockLocalHomePlatform.instance) {
+  /**
+   * Returns the static singleton instance, creating it if needed.
+   * @param resetState  Whether or not to force a new instance.  Useful for initializing tests.
+   */
+  public static getInstance(
+    resetState: boolean = false
+  ): MockLocalHomePlatform {
+    if (!MockLocalHomePlatform.instance || resetState) {
       MockLocalHomePlatform.instance = new MockLocalHomePlatform();
     }
     return MockLocalHomePlatform.instance;
   }
 
-  public getDeviceManager(): smarthome.DeviceManager {
-    return this.deviceManager;
-  }
-
-  public getLocalDeviceIdMap(): Map<string, string> {
-    return this.localDeviceIds;
+  private onNewDeviceIdRegistered(localDeviceId: string) {
+    this.newDeviceRegisteredActions.forEach(newDeviceRegisteredAction => {
```

Comment:
Does this need to be a list? (do we are of waiting for more than one registration at a time)

Reply:
Was able to remove this pattern and just return the local device id in `triggerIdentify`